GitVersionExe/Arguments.cs

```diff
@@ -33,5 +33,9 @@ public Arguments()
         public string UpdateAssemblyInfoFileName;
 
         public bool ShowConfig;
+
+        public bool IncludeUntrackedBranches;
```

Comment:
It's being used in the Context creation... last diff at: https://github.com/anobleperson/GitVersion/commit/539c5e493712492d9df34de0a2a6f9d5fbaf84b9